### PR TITLE
Casey boyland

### PR DESCRIPTION
**Pull request by caseyboyland**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,61 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/index.css">
```
Comment (katfsw) [inline index.html:6]: 🔥

File: index.html [hunk 2]
```diff
@@ -3,9 +3,61 @@
...
     <title>Document</title>
 </head>
 <body>
-    
+    <H1>Casey's Shop</H1>
+    <Section class='top-nav'>
+        <h3>Casey Boyland</h3>
+            <nav>
```
Comment (katfsw) [inline index.html:13]: awesome, you have a nav bar with "About", "Projects" and "Contact" - sweet sweet!

File: index.html [hunk 3]
```diff
@@ -3,9 +3,61 @@
...
+                <a href="http://about.html">About</a>
+                <a href="http://contact.html">Contact</a>
+                <a href="http://projects.html">Projects</a>
+            </nav>
+    </Section>
+
+    <Section class='my-name'>
+        <p>Casey Boyland</p>
+        <div class='my-name-link'>
+        <a href="http://projects.html">Projects</a>
```
Comment (katfsw) [inline index.html:24]: awesome job linking your projects 🔥 and displaying flex on them as well!
awesome job adding a button to link to the project page!

File: index.html [hunk 4]
```diff
@@ -3,9 +3,61 @@
...
+        <p>Custom Furniture</p>
+        </div>
+        <div class='desk'>
+        <img src='https://cb2.scene7.com/is/image/CB2/DrommenDeskSHS18_1x1'>
+        <p>Desks</p>
+        </div>
+    </section>
+
+    <footer>
+        <a href="http://contact.html">Contact Me</a>
```
Comment (katfsw) [inline index.html:59]: awesome job linking to a contact page!

Comment (katfsw) [reply index.html:59]: If you wanted to, you could add your social media somewhere as well !

**Review by katfsw — APPROVED** [review]
Awesome work Casey! Definitely looks like MVP to me! 🔥 Your code looks really clean and organized at each level, great job!

File: index.html (hunk 4 above)
Comment (caseyboyland) [reply index.html:59]: Thank you so much for the feedback Brandy! I appreciate you helping me out with everything.

**Pull request merged**